examples/routing/src/App.tsx

```diff
@@ -1,29 +1,19 @@
-import { Link, Redirect, Route, Switch } from "@prodo/route";
+import { Redirect, Route, Switch } from "@prodo/route";
 import * as React from "react";
-import { redirectHome, redirectUser } from "./actions";
-import { model } from "./model";
-
-const HomePage = model.connect(({ dispatch }) => () => (
-  <div>
-    HomePage
-    <button onClick={() => dispatch(redirectUser)("tom")}>Log In</button>
-  </div>
-));
-const UserPage = model.connect(({ watch, route, dispatch }) => () => (
-  <div>
-    Hello {watch(route.params.username)}.
-    <button onClick={() => dispatch(redirectHome)()}>Log Out</button>
-  </div>
-));
+import Home from "./pages/Home";
+import NotFound from "./pages/NotFound";
+import Profile from "./pages/Profile";
 
 const App = () => (
   <div className="app">
-    <Redirect to="/test-2" />
-    <Link to={{ path: "/test", params: { user: "tom" } }}>Click Me</Link>
+    <Route exact path="/">
+      <Redirect to="/home" />
+    </Route>
+
     <Switch>
-      <Route exact path="/user" component={UserPage} />
-      <Route exact path="/test" />
-      <Route component={HomePage} />
+      <Route exact path="/home" component={Home} />
+      <Route path="/:username" component={Profile} />
+      <Route component={NotFound} />
```

Comment:
If I go to /bla I don't hit Not Found. I guess that's expected since it could be a username? Should there be a check? same thing happens if I go to something more complex like /bla/bla/bla.
![Screenshot 2019-09-26 at 17 08 43](https://user-images.githubusercontent.com/1416759/65705426-55b6f480-e080-11e9-84ab-5bf8168d4f6e.png)

Reply:
oh thats a good point, I'll move user profile stuff under `/user`